homeassistant/components/huisbaasje/manifest.json

```diff
@@ -3,7 +3,7 @@
   "name": "Huisbaasje",
   "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/huisbaasje",
-  "requirements": ["huisbaasje-client==0.1.0"],
+  "requirements": ["energyflip-client==0.2.0"],
```

Comment:
Please fix the github link. It's a 404 now:

https://github.com/dennisschroer/energyflip-client/blob/850f64bc49525492debb7aba7db83be907cbaaa9/setup.py#L14